modules/machine-feature-aws-imds-options.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster_api_provider_configurations/cluster-api-config-options-aws.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine-feature-aws-imds-options_{context}"]
+= Amazon EC2 Instance Metadata Service configuration options
+
+You can restrict the version of the Amazon EC2 Instance Metadata Service (IMDS) that machines on {aws-first} clusters use.
+Machines can require the use of link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html[IMDSv2] (AWS documentation), or allow the use of IMDSv1 in addition to IMDSv2.
+
+////
+This is true but does not apply to TP clusters, reassess for Cluster API GA
+[NOTE]
+====
+To use IMDSv2 on AWS clusters that were created with {product-title} version 4.6 or earlier, you must update your boot image. 
+For more information, see "Updated boot images".
+====
+////
```

Comment:
These lines are commented out, but [that link](https://drive.google.com/file/d/1APbzIDc_Dc7GSD-8KTifQmPWfzMMxADj/view?usp=sharing) is still shown in the doc, maybe that link should also be commented out?

Reply:
Good catch!